trade-finance-manager-ui/server/controllers/case/cancellation/effective-from-date.controller.post.test.ts

```diff
@@ -23,6 +23,7 @@ jest.mock('../../../api', () => ({
 const dealId = 'dealId';
 const ukefDealId = 'ukefDealId';
 const mockUser = aTfmSessionUser();
+const defaultBackUrl = `/case/${dealId}/cancellation/bank-request-date`;
```

Comment:
I'm not sure if this should be here, but rather on line 156 since it's only used once, unless i'm missing something

Reply:
Same in other controllers

Reply:
I'm indifferent either way - I think there's some value in having a fixed constant at the top of the file here. But I've move it so it's inline on controllers where it's only used once (and kept it at the top on two controllers where it's used twice, but renamed it to `previousPage` so it looks a bit neater passed in).